src/cartesian/Area.tsx

```diff
@@ -533,8 +616,47 @@ function AreaImpl(props: Props) {
         dataKey: props.dataKey,
       }),
     ) ?? {};
+  const { height, width, left, top } = useOffset();
+  const updateId = useUpdateId();
+
+  if (layout !== 'horizontal' && layout !== 'vertical') {
+    // Can't render Area in an unsupported layout
+    return null;
+  }
+
+  if (chartName !== 'AreaChart' && chartName !== 'ComposedChart') {
+    // There is nothing stopping us from rendering Area in other charts, except for historical reasons. Do we want to allow that?
```

Comment:
I don't think for this release, maybe eventually everything becomes ComposedChart